specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -250,14 +248,9 @@ function TreeView<SCHEMA extends AnyTree>({
               return (
                 <div
                   className={`
-                  sticky top-0 whitespace-nowrap border border-transparent p-2
-                  ${index === 0 ? '-ml-2 rounded-bl pl-4' : ''}
-                  ${index + 1 === length ? '-mr-2 rounded-br pr-4' : ''}
-                  ${
-                    reduceTransparency || !supportsBackdropBlur
-                      ? 'bg-gray-100 dark:bg-neutral-900'
-                      : 'bg-gray-100/60 backdrop-blur-sm dark:bg-neutral-900/60'
-                  }
+                sticky top-0 whitespace-nowrap border border-transparent border-b-[color:var(--accent-color-300)] bg-[color:var(--background)] p-2 brightness-95 dark:brightness-105
```

Comment:
looks like formatting here is completely broken
unfortunately prettier doesn't fix it automatically
can you fix it please and be on the lookout for this in the future?

Mainly:
1. Lines should not be longer than 80 chars
2. The string should be indented (`sticky...` should be one more indentation level than `className={`, not one less)

Reply:
Also, the header has a totally different look now, it's just completely black?

![Screenshot 2023-07-03 at 19 08 00](https://github.com/specify/specify7/assets/40512816/e17567d4-15c6-40f1-8566-950f8563c52c)

not sure I like the new look. I think we can both get rid of the problematic blur effect, and have the header look somewhat similar to how it was before.

but I would defer to product engineers on this. @grantfitzsimmons

Reply:
yes we agreed with @grantfitzsimmons to make the header a solid color. The background of the header will be a slightly darker color than what the user choose for his normal SP background in light mode and slightly lighter color than what the user choose for his normal SP background in dark mode. We could choose to always have same color as a background but I thought I wasn't looking great when the user was customizing his b-g (we could increase the brightness more in dark-m and decrease more in light-m). 
The bottom border is also depending on the custom color 3 that the user is choosing. Again to have it harmonized with the user choices. 
I personally like the new look.

Reply:
Decision made with @grantfitzsimmons : 
- Increase brightness of the solid color b-g in dark-mode keep the solid color in light-mode